res/css/structures/_UserMenu.pcss

```diff
@@ -204,66 +204,3 @@ Please see LICENSE files in the repository root for full details.
         mask-image: url("@vector-im/compound-design-tokens/icons/qr-code.svg");
     }
 }
-
-.mx_UserMenu_CustomStatusSection {
```

Comment:
This css is not used anymore?

Reply:
It wasn't used since we removed custom status labs over a year ago, but finally cleaning it up here as it was referencing an x/close svg and updating it felt moot